01-intro.Rmd

```diff
@@ -16,15 +16,37 @@ First we will discuss the motivations behind the new data management and sharing
 
 So why is the NIH doing this? There are several reasons why sharing data can be beneficial to the scientific community.
 
-- **Supports transparency** - Sharing data provides more clarity about how studies are performed. 
--  **Encourages reproducibility and rigor** - Having the data accessible, allows others to try to reproduce study findings. This can further enable studies that may replicate or validate the initial findings with different data. 
+- **Supports transparency** - Sharing data provides more clarity about how studies are performed. Many scientists also believe in an ethical responsibility to study participants (@Bauchner_citation_2016).
+- **Encourages reproducibility and rigor** - Having the data accessible, allows others to try to reproduce study findings. This can further enable studies that may replicate or validate the initial findings with different data. 
 - **Supports multi-modal work** - When more data of various types are easily available it makes it easier for scientist to perform studies with multiple types of data. 
 - **More efficient and cost effective** - Some data are especially difficult or expensive to produce.
 - **Supports Researcher Inclusion** - Data generation can be especially difficult for those at institutes with less resources. Publicly available data can therefore be used by these researchers to better enable their participation. 
 - **Increased impact** - Papers that share their data in repositories appear to be cited more based on the study by @colavizza_citation_2020
 - **Increased collaboration opportunity** - Having data available can encourage other researchers to expand the research in a new direction or extend it further and they may reach out to collaborate.
 - **Data Citations** - Due to the importance of data generation and sharing to the NIH, data will now be seen as research product that demonstrates a contribution to the scientific community.
 
+## Key terms 
+
+<div class = "dictionary">
+
+**Data Management**
+
+The process of validating, organizing, protecting, maintaining, and processing scientific data to ensure the accessibility, reliability, and quality of the scientific data for its users. All data should be managed.
```

Comment:
have the word process and processing...(maybe want to use something else?) tricky though

Reply:
lifted straight from NIH - LOL but I will change it :)